scripts/unix_share_build.sh

```diff
@@ -110,7 +110,7 @@ function build_moab() {
     cd ${PLUGIN_ABS_PATH}
     mkdir -pv moab/bld
     cd moab
-    git clone https://bitbucket.org/fathomteam/moab -b Version5.1.0
+    git clone https://bitbucket.org/fathomteam/moab -b Version5.1.0 --depth 1
```

Comment:
Shouldn't we update this to 5.3.0 while we're at it?

Reply:
Thanks, I've updated to 5.3.0 now, I just wanted to compare like for like with the timings